docs/sources/plugins/plugin-signatures.md

```diff
@@ -8,9 +8,17 @@ aliases = ["/docs/grafana/latest/plugins/plugin-signature-verification"]
 
 Plugin signature verification (signing) is a security measure to make sure plugins haven't been tampered with. Upon loading, Grafana checks to see if a plugin is signed or unsigned when inspecting and verifying its digital signature.
 
-At startup, Grafana verifies the signatures of every plugin in the plugin directory. You can see the result of this verification for each plugin by navigating to **Configuration** -> **Plugins**.
+At startup, Grafana verifies the signatures of every plugin in the plugin directory. If a plugin is unsigned, then Grafana won't load or start it. You can see the result of this verification for each plugin by navigating to **Configuration** -> **Plugins**.
 
-> **Note:** If you're a plugin developer and want to know how to sign your plugin, refer to [Sign a plugin]({{< relref "../developers/plugins/sign-a-plugin.md" >}}).
+Grafana also writes an error message to the server log:
+
+```bash
+WARN[05-26|12:00:00] Some plugin scanning errors were found     errors="plugin '<plugin id>' is unsigned, plugin '<plugin id>' has an invalid signature"
+```
+
+> **Note:** All Grafana Labs authored backend plugins, including Enterprise plugins, are signed.
```

Comment:
This seems like an odd place for this statement. Isn't it assumed that we sign our own plugins? And if we're going to say it, I would recommend it be up in the prose, not in a note. Perhaps a paragraph leading in to the one in line 21.

Reply:
I agree. It feels like it should be implied that we sign our own plugins. Would it make more sense to invert this statement to list the Grafana Labs authored plugins that aren't (but should be) signed?

Reply:
I'd be happy to completely omit it if that works too?

>Would it make more sense to invert this statement to list the Grafana Labs authored plugins that aren't (but should be) signed?

Are there many of these? And will they exist like that for a long time?